cstring length doesn't work with cpp backend

``` nimrod
var x = "foo".cstring
echo x.len
```

I get the following error:

```
/home/deen/git/nimrod-unsorted/nimcache/min.cpp: In function ‘void minInit()’:
/home/deen/git/nimrod-unsorted/nimcache/min.cpp:98:35: error: ‘strlen’ was not declared in this scope
  LOC1 = nimIntToStr(strlen(x_86003));
```

In the cpp file string.h should be included, but I don't know where to add that.

--- a/compiler/ccgexprs.nim
+++ b/compiler/ccgexprs.nim
@@ -1286,7 +1286,7 @@ proc genArrayLen(p: BProc, e: PNode, d: var TLoc, op: TMagic) =
     if op == mHigh: unaryExpr(p, e, d, "($1Len0-1)")
     else: unaryExpr(p, e, d, "$1Len0")
   of tyCString:
-    if gCmd == cmdCompileToCpp: useStringh(p.module)
+    useStringh(p.module)
     if op == mHigh: unaryExpr(p, e, d, "(strlen($1)-1)")
     else: unaryExpr(p, e, d, "strlen($1)")
   of tyString, tySequence:

